app/models/goal.py

```diff
@@ -4,3 +4,5 @@
 
 class Goal(db.Model):
     goal_id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String)
+    tasks = db.relationship('Task', backref = 'objective', lazy = True)
```

Comment:
Do we want to store goals with a NULL title? Consider setting nullable=False to ensure every goal requires a title.